src/chrome/internal/stackTraces/stackTracePresentation.ts

```diff
@@ -0,0 +1,11 @@
+import { StackTracePresentationRow } from './stackTracePresentationRow';
+
+/** The stack traces we sent to the client will be represented by this classes and it is a combination of:
+ *    1. CallFrames with state information from the sync frames.
+ *    2. CodeFlowFrames without state information from async frames.
+ *    3. Labels that we use to [Show more frames] or [Frames skipped by smartStep], etc...
```

Comment:
Neither of those are labels that come from the DA, only the async frame labels come from the DA